netbox/dcim.py

```diff
@@ -250,6 +250,42 @@ def update_rack_group_by_id(self, rack_group_id, **kwargs):
         """
         return self.netbox_con.patch('/dcim/rack-groups/', rack_group_id, **kwargs)
 
+    def create_virtual_chassis(self, name, **kwargs):
+        """Create a new virtual chassis
+
+        :param name: Name of the virtual chassis
+        :param kwargs: Optional arguments
+        :return: netbox object if successful otherwise raise CreateException
+        """
+        required_fields = {"name": name}
+        return self.netbox_con.post('/dcim/virtual-chassis/', required_fields, **kwargs)
+
+    def get_virtual_chassis(self, **kwargs):
+        """Get all virtual chassis"""
+        return self.netbox_con.get('/dcim/virtual-chassis/', **kwargs)
+    
+    def update_virtual_chassis(self, virtual_chassis_name, **kwargs):
+        """Update virtual chassis by virtual chassis name
+
+        :param virtual_chassis_name: virtual chassis name to update
+        :param kwargs: requests body dict
+        :return: bool True if successful otherwise raise UpdateException
+        """
+        device_id = self.get_virtual_chassis(name=virtual_chassis_name)[0]['id']
```

Comment:
The variable name is device_id. Now I assume you would do a self.get_devices.

Reply:
Yep, you're right; I fixed it.

Reply:
Thanks